packages/expo-cli/src/accounts.ts

```diff
@@ -1,7 +1,7 @@
+import { ApiV2, User, UserManager } from '@expo/api';
 import chalk from 'chalk';
 import program from 'commander';
 import openBrowser from 'react-dev-utils/openBrowser';
-import { ApiV2, User, UserManager } from 'xdl';
 import { ApiV2Error } from 'xdl/build/ApiV2';
```

Comment:
```suggestion
```
❗Need to remove this and the `instanceof` check here: https://github.com/expo/expo-cli/blob/4a9cc924f1dd16fe42a3a1ae343b515fba7c435e/packages/expo-cli/src/accounts.ts#L325